.travis.yml

```diff
@@ -0,0 +1,12 @@
+sudo: required
+
+language: python
+python:
+  - "3.6.4"
+  
+jobs:
+  include:
+    - stage: install
+      script: test/travis/install.sh
+    - stage: lint
+      script: test/travis/lint.sh
```

Comment:
@svenmueller why are we using travis ? I thought we agreed on using Cloudbuilder for everything ops related or ?

Reply:
@anas-aso since this repo is public, we started using travis-ci.org for building it. so CI builds/logs are accessible.